_partials/_create-hypertable-blockchain.md

```diff
@@ -1,15 +1,20 @@
 
-## Optimize time-series data in hypertables
+## Optimize time-series data using hypertables
 
-Hypertables are the core of Timescale. Hypertables enable Timescale to work
-efficiently with time-series data. Because Timescale is PostgreSQL, all the
-standard PostgreSQL tables, indexes, stored procedures and other objects can be
-created alongside your Timescale hypertables. This makes creating and working
-with Timescale tables similar to standard PostgreSQL.
+$HYPERTABLE_CAPs are the core of $TIMESCALE_DB. $HYPERTABLE_CAPs enable $TIMESCALE_DB to work
+efficiently with time-series data. Because $TIMESCALE_DB is a $PG extension, all the
+standard $PG tables, indexes, stored procedures and other objects can be
+created alongside your $TIMESCALE_DB $HYPERTABLE. This makes creating and working
+with $HYPERTABLEs similar to standard $PG.
 
 <Procedure>
 
-1.  Create a standard PostgreSQL table to store the Bitcoin blockchain data
+1. Connect to your $SERVICE_LONG
```

Comment:
For reviewers, this example is updated to use CREATE TABLE In another PR: https://github.com/timescale/docs/pull/4089/files#diff-5953950de370451ab0d4f26a8324e4164fdf2bba054442ce754f957332b93b34. Please just ignore the code here.